common4j/src/main/com/microsoft/identity/common/java/commands/SilentTokenCommand.java

```diff
@@ -74,6 +72,7 @@ public AcquireTokenResult execute() throws Exception {
         final List<BaseController> controllers = getControllerFactory().getAllControllers();
 
         try (final Scope scope = SpanExtension.makeCurrentSpan(span)) {
+            Exception exceptionFromFirstController = null;
```

Comment:
>exceptionFromFirstController [](http://example.com/codeflow?start=22&length=28)

If the intent is to prioritize broker's exception for client, then I think it's better to capture that exception e.g. Exception exceptionFromBrokerMsalController;

then below, 
expcliit check on if (controller instanceof BrokerMsalController)


Instead of using controller ordering.

Reply:
+1

Reply:
The idea is to return error from first/primary controller so that client prioritize to handle that.
In case all controllers fail, we want the client to handle/recover from the error from primary/first controller if possible.